aspnet6/DummyResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace aspnet6;
+
+public readonly record struct DummyResponse(
+    String? Name, 
+    Boolean? IsActive, 
+    int? ThreadGroupActiveCount, 
```

Comment:
Missing an `isVirtual` field will affect the API response sizes, which breaks the control experiments and make the benchmark less useful.

Please help to enhance it.

Reply:
Added